about.css

```diff
@@ -1,6 +1,10 @@
 @import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
```

Comment:
According to the project requirements, you need to use the `Cocogoose` and `Lato` fonts, or at least fonts that are similar to them.

<img width="465" alt="Screen Shot 2022-07-27 at 9 25 32" src="https://user-images.githubusercontent.com/82426682/181272422-e98b25f3-e564-4ba6-b309-471d76e1ad08.png">

Please fix it.